apps/android/README.md

```diff
@@ -12,28 +12,53 @@ Prereqs:
 
 1. Make sure you have followed the [Getting Started](../../README.md) instructions to install packages and build the entire FluentUI React Native repository. I.e. from the root of the repo:
 
-```
-    yarn && yarn build
+```sh
+yarn
+yarn build
 ```
 
 2. Then go into `apps/android` folder and simply run the the following
 
-```
+```sh
 yarn android
 ```
 
-Troubleshooting
+## Dependencies
+
+Dependencies are managed by
+[`@rnx-kit/dep-check`](https://github.com/microsoft/rnx-kit/tree/main/packages/dep-check).
+If you're looking to upgrade `react-native`, look for the `rnx-kit` section in
+`/apps/android/package.json`:
+
+```json
+{
+  ...
+  "rnx-kit": {
+    "reactNativeVersion": "^0.63",
+    "kitType": "app",
+    "bundle": {
+  ...
+}
+```
+
+Bump `reactNativeVersion`, and run `yarn rnx-dep-check --write`. This command
+will ensure that all relevant packages are bumped correctly.
+
+You can read more about this tool here:
+[`@rnx-kit/dep-check` design document](https://github.com/microsoft/rnx-kit/blob/main/packages/dep-check/DESIGN.md)
+
+## Troubleshooting
```

Comment:
What's up with the syntax highlighting here with GitHub? All of the characters have black boxes behind them for me

Reply:
Yeah, I see it too. You can try rich diff instead:

![image](https://user-images.githubusercontent.com/4123478/123478245-774f6200-d5ff-11eb-9a5f-9353d758cab5.png)

Reply:
Rich diff shows things as fine, which is why I figured it's ok.